.changeset/green-jobs-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": minor
+---
+
+Add suport for generic update operators
```

Comment:
I would expand this changeset as it could be used to understand what to expect from the new syntax with an example like:
```
 mutation {
            updateMovies(update: { name: { set: "The Matrix" } }) {
                movies {
                    id
                    name
		}
       }
}
```